scripts/build.py

```diff
@@ -161,7 +163,11 @@ def generate_app_yaml(is_prod_mode=False):
             content = content.replace(
                 file_path,
                 prod_file_prefix + file_path)
-    if not is_prod_mode:
+    # The version: default line is required to run jobs on a local server (
+    # both in prod & non-prod mode). This line is not required when app.yaml
```

Comment:
here and around line 1263, change the mode to env. (Let do it in any other PR as is very minor and currently I need this PR for release.)

Reply:
Done: https://github.com/oppia/oppia/pull/8568